src/containers/Card.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div
+    class="card"
+    @click="onClick"
+  >
+    <image-item :src="card.src" />
+    <div class="header">
+      <h1 class="title">
+        {{ card.title }}
+      </h1>
+      <p class="year">
+        {{ card.year }}
+      </p>
+    </div>
+    <p>{{ card.genre }}</p>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from 'vue';
+import { CardType } from '@/types';
+
+export default defineComponent({
+  name: 'Card',
+  props: {
+    card: { type: Object as PropType<CardType>, required: true },
+    onClick: { type: Function as ()=> void, required: true },
```

Comment:
Events are how we communicate to parent components in Vue. Please have a look at https://v3.vuejs.org/guide/component-custom-events.html#event-names